src/desktop/lib/palette.styl

```diff
@@ -0,0 +1,69 @@
+THEME = json('../../../node_modules/@artsy/palette/dist/tokens.json', { hash: true })
+
+// Access themed values
+
+getFontFamily(name)
+  value = THEME.fonts[name]
+  if value == null
+    error('cannot find: ' + name)
+  value
+
+getFontSize(name)
+  value = THEME.fontSizes[name]
+  if value == null
+    error('cannot find: ' + name)
+  value
+
+getLineHeight(name)
+  value = THEME.lineHeights[name]
+  if value == null
+    error('cannot find: ' + name)
+  value
+
+getLetterSpacing(name)
+  value = THEME.letterSpacings[name]
+  if value == null
+    error('cannot find: ' + name)
+  value
+
+getColor(name)
+  value = THEME.colors[name]
+  if value == null
+    error('cannot find: ' + name)
+  value
```

Comment:
I wanted to just namespace these functions like `palette.color` but there's a bug about this open in stylus since like 2014.